src/styles.scss

```diff
@@ -6,3 +6,31 @@ body{
     background-color: rgb(239, 216, 202);
 
 }
+
+input {
+    width: 200px;
+    padding: 0.5rem;
+    font-size: 1rem;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+
+    &:focus {
+      outline: none;
+      border-color: #004b9b; 
+    }
+}
+
+button {
```

Comment:
please give on element button
class name and use in scss
its more common

Reply:
fixed